packages/ui-react/src/components/Player/Player.tsx

```diff
@@ -72,7 +72,14 @@ const Player: React.FC<Props> = ({
   const handleComplete = useEventCallback(onComplete);
   const handleUserActive = useEventCallback(onUserActive);
   const handleUserInactive = useEventCallback(onUserInActive);
-  const handleFirstFrame = useEventCallback(onFirstFrame);
+  const handleFirstFrame = useEventCallback(() => {
+    // a11y: auto focus to play/pause button
+    const playPauseButton = document.querySelector('.jw-controlbar [aria-label="Play"], .jw-button-container [aria-label="Pause"]') as HTMLDivElement | null;
+
+    playPauseButton?.focus();
+
+    onFirstFrame?.();
+  });
```

Comment:
Does this also work (as expected) for the inline player variant?

Reply:
I hadn't checked (apologees!) but I did just now: works!